### ISSUE TITLE
The request header REQUEST_URI contains PII or secrets that should be stripped via filter_parameters

### ISSUE THREAD

**Issue by jbockler** [appsignal/appsignal-ruby#1038]
## Describe the bug

Eventhough a GET parameter is configured in filter_parameters due to it containing PII or a secret API token for example, the REQUEST_URI request header is not getting filtered and is leaking this information to Appsignal.

Of course i could simply remove the REQUEST_URI from `request_headers` config, but it is not stated in the docs anywhere
and is easy to oversee. On the other hand from description of the `filter_parameters` i would have assumed that this will get filtered too.

## To Reproduce

- Using AppSignal for Ruby gem version 3.5.5
- In my app using rails 7.1.2
- With this code:
  ```
  # appsignal.yml:
  default: &defaults
    push_api_key: "foo"
    name: "MyApplication"
    filter_parameters: ["api_token"]

  development:
    <<: *defaults
    active: true

  production:
    <<: *defaults
    active: true
  ```
- Doing Request to a route and add `?api_token=staysecret` and the request should fail
- I see in AppSignal in the "Parameters" section the parameters correctly as "[FILTERED]"
- In the "Environment" section i see the whole REQUEST_URI with `/sample?api_token=staysecret`

**Comment by luismiramirez** [comment]
Hi @jbockler thanks for the report. We'll keep this issue updated while we work on it.

**Comment by luismiramirez** [comment]
We've taken the decision to remove `REQUEST_URI` from the default headers in favor of `REQUEST_PATH.`

`REQUEST_PATH` doesn't include query params that might contain sensitive information. Sanitizing the contents of `REQUEST_URI` would be inaccurate, and the door to sensitive data would never be completely closed.

Until the new version of the integration is released, we recommend you manually config `request_headers` and not include `REQUEST_URI`

**Comment by jbockler** [comment]
Alright, thank you for looking into this issue @luismiramirez !

**Comment by luismiramirez** [comment]
@jbockler The fix is now published on version 3.5.6. Again, thank you for the detailed report!
